nima/webserver/webserver/src/main/java/io/helidon/nima/webserver/http/ErrorHandlers.java

```diff
@@ -155,11 +156,16 @@ private void handleRequestException(ConnectionContext ctx,
         }
         boolean keepAlive = e.keepAlive();
         if (keepAlive && !request.content().consumed()) {
-            try {
-                // attempt to consume the request entity (only when keeping the connection alive)
-                request.content().consume();
-            } catch (Exception ignored) {
-                keepAlive = request.content().consumed();
+            if (request.headers().contains(Http.Header.EXPECT)) {
```

Comment:
What if `100 continue` was already sent?

Reply:
Nice catch thx!

Reply:
Tested [here](https://github.com/helidon-io/helidon/pull/5965/files#diff-981d3f532e3423a0713eec368def68638214b4c398c51f12a4d767aab990c497R225-R246) and [here](https://github.com/helidon-io/helidon/pull/5965/files#diff-9055aeb73e7697f0a137365e717a256d6201c6d459ce29d5bbccfd5145733061R248-R271) for immediate continue